source/includes/language-compatibility-table-java.rst

```diff
@@ -180,11 +211,12 @@
      -
 
    * - 3.0
+     -
      -
      - ✓
      - ✓
      - ✓
      - ✓
      -
 
-.. [#lts-note] In the future, only LTS Java versions will be explicitly listed.
+.. [#virtual-threads-note] This driver version is compatible with Java 21 if your application uses virtual threads.
```

Comment:
I think this should be: 

> This driver version is not compatible with virtual threads.

since the note is for older releases that have not been tested with virtual threads, not newer ones that have.

Reply:
If we leave out the `Java 21` bit as I suggest, it doesn't have to be updated for Java 25 :)